projects/vdk-core/src/vdk/internal/plugin/plugin.py

```diff
@@ -50,6 +52,8 @@ def load_plugins_from_setuptools_entrypoints(self) -> None:
 
         try:
             self.__plugin_manager.load_setuptools_entrypoints(self.__group_name)
+        except ImportError as e:
```

Comment:
At first glance assuming that each import error must be a user error seems strange. There could be a bug in the plugin itself that results in an Import error. 

Perhaps we discussed that (I have vague memories) but if we had I can't recall the reasoning for why we want this to be user error. What is the reason ?